setup.py

```diff
@@ -3,11 +3,8 @@
 
 _version_file = 'hooked/version.py'
 
-if sys.version_info.major < 3:
-    execfile(_version_file)
-else:
-    with open(_version_file) as vf:
-        exec(vf.read())
+with open(_version_file) as vf:
+    exec(vf.read())
```

Comment:
Mm, then I think that this change is fine.

It's not within a function, and I doubt that our `setup.py` will get so complicated that we'll one-day need to wrap this in a function... YAGNI!